docs/developer/users.mdx

```diff
@@ -68,14 +79,18 @@ As a result, we get the data of the newly created user:
       "errors": [],
       "user": {
         "email": "customer@example.com",
-        "isActive": true
+        "isActive": true,
+        "isConfirmed": false
       }
     }
   }
 }
 ```
 
-The `isActive` flag informs that the account is active and the user can log in to their account.
+The `isActive` flag informs that the account is active. If `active` is set to `False` the user can never log in.
```

Comment:
Maybe you can connect `isActive` and `isConfirmed` with the fields on `User` type from `api-reference`?